src/components/Inputs/BirthdateInput/BirthdateInput.js

```diff
@@ -99,6 +101,8 @@ const PrivateBirthdateInput = (props) => {
           !!err ? 'BirthdateInput TextInput Error' : 'BirthdateInput TextInput'
         }
         type="text"
+        inputMode={INPUT_MODES.NUMERIC}
```

Comment:
I really like how you leveraged DI on zip and birthdate here @zach-ethos 👍 nice job! I think it's flexible but still super simple in terms of understandability

Reply:
Can you explain what you mean by DI here? (IRL maybe)